tests/unit/index.js

```diff
@@ -65,7 +65,7 @@ describe('X25519KeyPair', () => {
 
       const result = key.verifyFingerprint(fingerprint);
       expect(result.valid).to.be.true;
-      expect(result.error).to.not.exist;
+      // expect(result.error).to.not.exist;
```

Comment:
Why is this commented out?  Seems it should be enabled or removed.

Reply:
Fixed.